backend/src/game/constants.py

```diff
@@ -7,6 +7,8 @@
 DEFAULT_HEALTH = 10
 ZERO = 0
 CARD_STORE_SIZE_LIMITER = 3
+DEFAULT_RE_ROLL_COUNT = 2
+DEFAULT_DICE_COUNT = 6
```

Comment:
I was already overruled by the professor on this, but I want to make a note that I believe that all things dice should reside in the dice file. These I feel should be class-level variables.

https://www.digitalocean.com/community/tutorials/understanding-class-and-instance-variables-in-python-3

Reply:
Appreciate the relentlessness, but still overruled. These should not be owned by the Dice class or the DiceHandler class. They shouldn't even be used inside of the Dice or DiceHandler class (which will be refactored later).
The engine responsible for using DiceHandler should pass these values in as input params for a given player.
something like:
dice_handler(DEFAULT_RE_ROLL_COUNT, DEFAULT_DICE_COUNT)
or if the player has special cards, maybe dice_handler(DEFAULT_RE_ROLL_COUNT+1, DEFAULT_DICE_COUNT)
The responsibility of knowing the rulebook should not be in Dice or DiceHandler is the point.

Reply:
@cvennel Actually we can do this refactor now, save us effort later. Please add these two constants as input params. I *think* dice_count can just come in to start_turn(), but reroll_count should be class level? Not sure though